tests/integration/test_charm.py

```diff
@@ -15,8 +15,8 @@
 
 logger = logging.getLogger(__name__)
 
-METADATA = yaml.safe_load(Path("./metadata.yaml").read_text(encoding="utf-8"))
-APP_NAME = METADATA["name"]
+CHARMCRAFT = yaml.safe_load(Path("./charmcraft.yaml").read_text(encoding="utf-8"))
+APP_NAME = CHARMCRAFT["name"]
```

Comment:
just a comment, this could be anything, no need to read the charmcraft.yaml file

Reply:
Yeah, true. I don't think it's a big problem but something to bear in mind, possibly for future PRs.